src/Workspaces/CSharp/Portable/EmbeddedLanguages/VirtualChars/CSharpVirtualCharService.cs

```diff
@@ -240,7 +274,7 @@ public override bool TryGetEscapeCharacter(char ch, out char escapedChar)
         }
 
         private bool TryAddMultiCharacterEscape(
-            ArrayBuilder<VirtualChar> result, string tokenText, int offset, int index, char character)
+            ArrayBuilder<(char ch, TextSpan span)> result, string tokenText, int offset, int index, char character)
```

Comment:
❔ Why is this `char ch` instead of `Rune ch`? Just trying to figure out what this code is supposed to be doing.

Reply:
because this is the pass where we pick out the char pieces, to eventually try to make into runes.  i.e. you might have `\\ud83d\ude0a`, i.e. an actual escaped surrogate char and an underlying surrogate char.  We collect these separately into their char values, then go over another pass to make the runes.

Reply:
will doc this.